corehq/apps/ota/admin.py

```diff
@@ -35,8 +35,16 @@ def clean(self):
                 raise ValidationError("You must specify both {} and {}."
                                       .format(version_min, version_max))
 
-        if data['cc_all_versions'] and data['app_all_versions']:
-            raise ValidationError("You cannot apply a measure to all cc and app versions.")
+        application_measures = ['app_reinstall_ota', 'app_reinstall_local', 'app_update']
+        commcare_measures = ['cc_reinstall', 'cc_update']
```

Comment:
should these be in a constant somewhere?

Reply:
These correspond to the `Measure`s in the other file in this PR.  I thought about making each measure be responsible for its own validation, or categorizing them and mapping validation that way, but figured that would be a little over-architected at this scale and decided to keep it simpler, if slightly repetitive.